.travis.yml

```diff
@@ -5,7 +5,21 @@ cache:
   directories:
     - "node_modules"
 script:
-  - npm run lint
-  - npm run jest -- --coverage
-  - npm run test:coverage
-after_success: npm run coveralls
\ No newline at end of file
+  - yarn lint
+  - yarn jest -- --coverage
+  - yarn test:coverage
```

Comment:
Does that work without problems? I tend to still use `npm run XYZ` in config files.

Reply:
Surprisingly, it does. TavisCI Node 8 image comes with yarn preinstalled. You can see the [CI logs](https://travis-ci.org/twitch-apis/twitch-js/builds/339967962) for confirmation.